test/focus.jest.js

```diff
@@ -59,4 +59,45 @@ describe('Page.focus', function() {
     expect(await page.$eval('#i1', e => e.value)).toBe('First');
     expect(await page.$eval('#i2', e => e.value)).toBe('Last');
   });
+  it('should traverse focus in all directions', async function({page}) {
+    await page.setContent(`<input value="1"><input value="2"><input value="3">`);
+    await page.keyboard.press('Tab');
+    expect(await page.evaluate(() => document.activeElement.value)).toBe('1');
+    await page.keyboard.press('Tab');
+    expect(await page.evaluate(() => document.activeElement.value)).toBe('2');
+    await page.keyboard.press('Tab');
+    expect(await page.evaluate(() => document.activeElement.value)).toBe('3');
+    await page.keyboard.press('Shift+Tab');
+    expect(await page.evaluate(() => document.activeElement.value)).toBe('2');
+    await page.keyboard.press('Shift+Tab');
+    expect(await page.evaluate(() => document.activeElement.value)).toBe('1');
+  });
+  // Chromium and WebKit both have settings for tab traversing all links, but 
+  // it is only on by default in WebKit.
+  it.skip(!MAC || !WEBKIT)('should traverse only form elements', async function({page}) {
```

Comment:
safari has this option:
<img width="326" alt="image" src="https://user-images.githubusercontent.com/4624233/87953368-cee46580-ca5f-11ea-8c13-982daa86baa9.png">

When its off, tab traversal skips all buttons and links. But you can still traverse every item with Option+Tab. I added a test to make sure we don't regress this.